app/model/package.scala

```diff
@@ -44,6 +44,17 @@ object FileCheckingResult {
 }
 
 case class ValidFileCheckingResult(location: S3ObjectLocation) extends FileCheckingResult
+
 sealed trait InvalidFileCheckingResult extends FileCheckingResult
 case class FileInfectedCheckingResult(location: S3ObjectLocation, details: String) extends InvalidFileCheckingResult
 case class IncorrectFileType(location: S3ObjectLocation, typeFound: MimeType) extends InvalidFileCheckingResult
+
+case class AllowedFileTypes(serviceName: String, allowedFileTypes: List[String])
+
+case class ConsumingServicesConfiguration(serviceConfigurations: List[AllowedFileTypes]) {
+  def allowedFileTypes(consumingService: String): List[String] =
+    serviceConfigurations
+      .find(_.serviceName == consumingService)
```

Comment:
I would think about using map [service,allowed file types] but anyway for the number of services we might have it doesn't matter.

Reply:
@tomasz-rosiek My thinking on this was that this model would be easy to extend with file size when we add that in, but could be pre-emptive on my part.